### ISSUE TITLE
[Replay Bug]: Failure to render websites created with Shorthand.com

### ISSUE THREAD

**Issue by Shrinks99** [webrecorder/replayweb.page#272]
### ReplayWeb.page Version

v1.8.14

### What did you expect to happen? What happened instead?

Websites created with https://shorthand.com/ don't seem to load properly in ReplayWebpage.  The files with the requisite content can be found in the WACZ after archiving with ArchiveWebpage, but they seem to generate 404 errors in the console on replay.

Interestingly, this is a good example of an issue that should be caught by our automated QA system as specced in https://github.com/webrecorder/browsertrix-cloud/issues/1435!

The console error in question:
```
Element for Story https://cambridge-university-press-asses.shorthandstories.com/five-books-on-climate-action-for-cop28 could not be found.
```

Example archive: [cambridge.wacz.zip](https://github.com/webrecorder/replayweb.page/files/13654058/cambridge.wacz.zip)

### Step-by-step reproduction instructions

1. Visit the link https://www.cambridge.org/news-and-insights/five-books-on-climate-action-for-cop28 and archive it to a new archiving session in ArchiveWebpage
2. Once all the network requests have been successfully received, attempt to view and replay the page.
3. The main Cambridge.org site nav and footer should load, but the page content won't render properly
4. Unzip the WACZ and the WARC.GZ files and search for any of the text on the original page that is missing on replay, it should be found within the `cambridge-university-press-asses.shorthandstories.com > five-books-on-climate-action-for-cop28 > assets > embed.js` file

**Comment by ikreymer** [comment]
Should have a fix for this for 2.0.2

**Comment by ikreymer** [comment]
Fixed in 2.0.2
